### DISCUSSION TITLE
ARM runners

### DISCUSSION THREAD

**Discussion by lmilbaum** [npalm/terraform-aws-gitlab-runner#1114] [Ideas]
Ability to run workloads on ARM based runners

**Comment by kayman-mk** [comment]
Should be possible with an ARM instance, the ARM version of the AMI and the ARM version of the Runner.

What did you try so far?

**Reply by lmilbaum** [reply]
I've tried the following options:
1. Replacing the runner and worker instance types and AMIs
2. Replacing only the runner instance type and AMI - that is not good because I want to be a container image on ARM
3. Replacing only the worker instance type and AMI 

Every option didn't work. I didn't invest cycles yet on how to debug the runner and worker, hence I don't know what failed

**Reply by lmilbaum** [reply]
Is there a need to use an ARM based gitlab runner while the workloads are actually running on the workers?

**Reply by lmilbaum** [reply]
NM. At last, found a configuration which worked.

**Reply by kayman-mk** [reply]
Could you share the config you have found, please? Could be helpful for others as well.
